lib/srv/discovery/access_graph.go

```diff
@@ -366,7 +366,7 @@ func (s *Server) initializeAndWatchAccessGraph(ctx context.Context, reloadCh <-c
 	}()
 
 	currentTAGResources := &aws_sync.Resources{}
-	ticker := time.NewTicker(15 * time.Minute)
+	ticker := time.NewTicker(s.PollInterval)
```

Comment:
What does this default to?

Reply:
5 minutes, per https://github.com/gravitational/teleport/blob/master/lib/srv/discovery/common/watcher.go#L40

Reply:
So this will worsen the problem then, no? Unless it's set to a higher interval elsewhere in the config, which seems error-prone.

Reply:
I think the best approach here is to add another option dedicated to access graph to avoid messing around with real discovery polling interval.

If you don't want to you can always max(15m, poll interval)

Reply:
That sounds good to me.

Reply:
Changing the primary default poll interval to suit the needs of both the access graph and the other discovery functions does seem problematic operationally. Although having just one poll interval to configure would be less confusing. But for now I'll just add an extra poll interval setting in the access graph config.